### ISSUE TITLE
Issue with vk videos

### ISSUE THREAD

**Issue by stl3** [omega-gg/MotionMonkey-dev#5]
Using other browsers (ffox, floorp, brave, operagx, edge, vivaldi, thorium I am able to access the vk video and play it by copying the url from MotionMonkey. The issue is playing the video from inside MotionMonkey itself, I am unable to play it. I have double checked and disabled pihole and I am still unable to get MotionMonkey to play it. 
Steps to reproduce:
1. Search for "Godot complete" (default search used)
2. Click on the third video titled "Godot 4 - A Complete Lighting Guide for 3D Games.."
3. Clicking on play does nothing (spinning loading circle about 2-3s and then goes away).

**Comment by 3unjee** [comment]
Can you provide the URL that fails ?

**Comment by stl3** [comment]
This was the specific url that was failing: [https://vk.com/wall-226892821_394](https://vk.com/wall-226892821_394)

**Comment by 3unjee** [comment]
The vk website is not supported. Try to do a "More Like This" and find equivalents elsewhere.

**Comment by stl3** [comment]
No worries, it was only because it came up in the search results so I thought it should play, will try as you suggested but the video is also available alternatively on yt so it's no biggie. 
So in summary, even if it might appear in a search results with thumbnail, it might not always be supported (is it an aftereffect of the parsing of all results from ddg as a whole for instance)? Sites that are technically able to be loaded by yt-dlp should be no problem though right?

**Comment by 3unjee** [comment]
Yes, it's a DuckDuckGo search, most of what you'll see should be playable, however there might be some exceptions.
